UnanimusProject/build.gradle

```diff
@@ -15,7 +15,10 @@ android {
     compileSdkVersion = 'android-22'
     buildToolsVersion = '22.0.1'
     defaultConfig {
-        applicationId "com.sjgilbert.unanimus"
+        applicationId 'com.sjgilbert.unanimus'
+    }
+    lintOptions {
+        disable 'IconMissingDensityFolder'
```

Comment:
We should at some point add the missing `UnanimusProject/res/drawable-xxhdpi` and `UnanimusProject/res/drawable-xxxhdpi` resource directories.